src/current/v23.2/physical-cluster-replication-overview.md

```diff
@@ -0,0 +1,132 @@
+
+In a disaster recovery scenario, you can [_cut over_]({% link {{ page.version.version }}/cutover-replication.md %}) from the unavailable primary cluster to the standby cluster. This will stop the replication stream, reset the standby cluster to a point in time where all ingested data is consistent, and mark the standby as ready to accept application traffic.
+
+{% comment %}A lot of links at the end of this section to general CV docs{% endcomment %}
+
+## Use cases
+
+You can use physical cluster replication in a disaster recovery plan to:
+
+- Meet your RTO (Recovery Time Objective) and RPO (Recovery Point Objective) needs. Physical cluster replication provides lower RTO and RPO than [backup and restore](backup-and-restore-overview.html). {% comment %}link here to upcoming DR tool comparative table{% endcomment %}
+- Automatically replicate everything in your primary cluster to recover quickly from a control plane or full cluster failure.
+- Cover different regions when you cannot use individual [multi-region clusters](multiregion-overview.html). Primary and standby clusters can exist in different regions.
+- Avoid conflicts in data after recovery because the replication completes to a transactionally consistent state as of a certain point in time.
+
+## Features
+
+- **Asynchronous byte-level replication**: When you initiate a replication stream, it will replicate byte-for-byte all existing user data and associated metadata (e.g., zone configurations) in the primary cluster to the standby cluster asynchronously. From then on, it will continuously replicate user data and metadata in the primary to the standby. Physical cluster replication will automatically replicate changes related to operations such as, schema changes, user and privilege modifications, and zone configuration updates without any manual work.
+- **Transactional consistency**: When you cut over to the standby, you can specify `LATEST`, a certain point of time in the past, or future. The standby cluster will be in a transactionally consistent state as of a certain point in time after the cutover process.
+- **Maintained/improved RPO and RTO**: Depending on workload and deployment configuration, [replication lag]({% link {{ page.version.version }}/physical-cluster-replication-technical-overview.md %}) between the primary and standby is generally in the tens of seconds range. The cutover process from the primary cluster to the standby should typically take 5 minutes or under when completing a cutover to the latest replicated time using `LATEST`.
+- **Cutover to a timestamp in the past**: In the case of logical disasters or mistakes, you can cut over from the primary to the standby cluster to a timestamp in the past. This means that you can return the standby to a timestamp before the mistake was replicated to the standby. You can also configure the `WITH RETENTION` option to control how far in the past you can cut over to. {% comment %}link retention option here{% endcomment %}
```

Comment:
In the Tasks page you also cover restoring to a point in time in the future. Is that worth mentioning here too?

Reply:
I have mentioned here on the same bullet, hope that sounds OK.